test/zksync-tests/UniversalRouterIntegration.test.ts

```diff
@@ -0,0 +1,350 @@
+import { UniversalRouter, Permit2, ERC721 } from '../../typechain';
```

Comment:
Am I understand correct that it's the test rewritten from the foundry?

Reply:
Yes. The foundry test `UniversalRouter.t.sol` was split in two separate hardhat tests `UniversalRouter.test.ts` and `UniversalRouterIntegration.test.ts`

Reply:
Then where is the `[UniversalRouter.test.ts](https://github.com/AntonD3/universal-router/blob/v1.2.2/test/integration-tests/UniversalRouter.test.ts)` now?

Reply:
Sorry for misconfuse. 
1. integration-tests/UniversalRouter.test.ts was adapted as zksync-tests/UniversalRouterIntegration.test.ts
2. UniversalRouter.t.sol was adapted as zksync-tests/UniversalRouter.test.ts